src/test/shared/sam/debugger/samDebugConfigProvider.test.ts

```diff
@@ -246,30 +286,76 @@ describe('AwsSamDebugConfigurationProvider', async () => {
                     //projectRoot: 'root as in beer'
                 },
             })
-            assert.strictEqual(resolved!.name, name)
+            // TODO: why not respect caller-chosen name?
+            assert.strictEqual(resolved!.name, 'SamLocalDebug')
         })
-        it('target=code', async () => {
-            const debugConfig = {
+
+        it('target=code: javascript', async () => {
+            const appDir = path.join(getProjectDir(), 'integrationTest-samples/js-manifest-in-root/')
```

Comment:
This change appears to introduce component tests that access samples intended for integration tests. 

Is there a risk that the integration test samples could get changed in the future without realizing there is an impact to tests like these? Is there value in classifying the sample content?

Reply:
I think the "integ-samples" folder should be renamed to e.g. `test-fixtures`.

If they are changed, tests need to be updated. No escaping that...